dask/dataframe/io/tests/test_sql.py

```diff
@@ -332,3 +335,98 @@ def test_no_character_index_without_divisions(db):
     # attempt to read the sql table with a character index and no divisions
     with pytest.raises(TypeError):
         read_sql_table("test", db, npartitions=2, index_col="name", divisions=None)
+
+
+def test_to_sql():
+    df_by_age = df.set_index("age")
+    df_appended = pd.concat([df, df,])
+
+    # Set this to False to use a sqlite DB file under the current directory, and not delete it (for easier debugging
+    # after tests have run)
+    rm_tmpdir = True
+    if rm_tmpdir:
+
+        @contextmanager
+        def tmp():
+            with tmpfile() as f:
+                yield "sqlite:///%s" % f
+
+    else:
+        from pathlib import Path
+
+        path = Path.cwd() / "tmp" / "sqlite"
+        path.parent.mkdir(exist_ok=True, parents=True)
+        uri = f"sqlite:///{path}"
+
+        @contextmanager
+        def tmp():
+            if path.exists():
+                path.unlink()
+            yield uri
+
+    for npartitions in [1, 2, 10]:
```

Comment:
This could be parameterized by pytest. But what's the goal of testing multiple partitions? I don't see anything that's sensitive to the number of partitions.

Testing 1 and 2 partitions is probably sufficient.

Reply:
yea I mimicked other tests that do `1` and `2` partitions (testing multiple partitions at least a bit seems impt)

`10` was probably overkill looking for edge cases, removed

Reply:
following up: I parameterized the test case by `npartitions`; much better, thx